### DISCUSSION TITLE
Come trovo l'ID del conto corrente?

### DISCUSSION THREAD

**Discussion by DifferentSailing** [fattureincloud/api#408] [Q&A] [answered]
Ciao a tutti! 

Premetto che lavoro in ambito automation no-code, non ho esperienza di programmazione. 

Ho integrato fatture in cloud con Zapier, ma ho bisogno del ID del conto corrente per creare automaticamente fatture in status "paid". Non riesco a capire come fare. 

C'è qualcuno che mi può aiutare? 

Grazie e a presto!

**Comment by emajo** [comment, marked as answer]
Ciao @DifferentSailing, abbiamo giusto implementato ieri una funzionzionalità per facilitare il settagggio di alcuni campi, dovrebbe infatti comparire la lista dei conti di pagamento cliccando sul campo `payments_list.payment_account.id`.

Ci faccia sapere se le risulta.
grazie,
Emanuele.

**Comment by DifferentSailing** [comment]
Si ha funzionato @emajo ! Grazie mille e buona giornata,
Federico.
